### PR TITLE
change to quay production

### PR DESCRIPTION
**Pull request by hemalathagajendran**
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## How risky is this change?
- [x] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Review by Jainbrt — APPROVED** [review]

File: generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: ibm-spectrum-scale-csi-operator
       containers:
       - name: operator
-        image: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-operator@sha256:c292fa07a03f5accac554a5f366b7241c31dbb85fc4617c0fe2428cf103fce56
```
Comment (Jainbrt) [inline generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml:36]: I think we should not deliver this yaml as a part of 2.13 and beyond.

**Review by saurabhwani5 — COMMENTED** [review]
@hemalathagajendran , Images are getting pulled properly 
```
[root@saurabh56-x-master new]# oc describe pod | grep quay  | grep Image:
    Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver@sha256:6cf6a3818f48c2bcb5460c4902c5d44ad73c415a5f2980a374062a4bb18840ea
    Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver@sha256:6cf6a3818f48c2bcb5460c4902c5d44ad73c415a5f2980a374062a4bb18840ea
    Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator@sha256:c292fa07a03f5accac554a5f366b7241c31dbb85fc4617c0fe2428cf103fce56
```

**Review by badri-pathak — APPROVED** [review]

File: generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml (hunk 1 above)
Comment (hemalathagajendran) [reply generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml:36]: Deleted the manifests for OCP+RHEL

**Comment by hemalathagajendran** [comment]
> @hemalathagajendran , Images are getting pulled properly
> 
> ```
> [root@saurabh56-x-master new]# oc describe pod | grep quay  | grep Image:
>     Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver@sha256:6cf6a3818f48c2bcb5460c4902c5d44ad73c415a5f2980a374062a4bb18840ea
>     Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver@sha256:6cf6a3818f48c2bcb5460c4902c5d44ad73c415a5f2980a374062a4bb18840ea
>     Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator@sha256:c292fa07a03f5accac554a5f366b7241c31dbb85fc4617c0fe2428cf103fce56
> ```

Thanks @saurabhwani5

**Review by badri-pathak — CHANGES_REQUESTED** [review]
We might have to look at the manifests generate function also. Because since we have removed rhel-yaml file, but manifests gen would have regenerate it.

https://github.com/hemalathagajendran/ibm-spectrum-scale-csi/blob/5052e84970423ffe2b0f835fb8d60a22578e1a4d/operator/Makefile#L87

**Comment by hemalathagajendran** [comment]
> We might have to look at the manifests generate function also. Because since we have removed rhel-yaml file, but manifests gen would have regenerate it.
> 
> https://github.com/hemalathagajendran/ibm-spectrum-scale-csi/blob/5052e84970423ffe2b0f835fb8d60a22578e1a4d/operator/Makefile#L87

Yes we have to make changes in makefile. But if we do that we have to change the sha's and we already signed off so we decided do that in 2.14.

**Comment by hemalathagajendran** [comment]
> @hemalathagajendran means this pr is good to merge and you will raise another pr in dev branch to remove ocp with rhel changes from manifests generate function , right ?

yes

**Review by badri-pathak — APPROVED** [review]

**Pull request merged**